### ISSUE TITLE
Changing profile picture

### ISSUE THREAD

**Issue by ritvij14** [garimasingh128/gitsetgo#85]
**Is your feature request related to a problem? Please describe.**
Many times people do not use their real photos on their GitHub profile, but they would want to show who they are using a real photo for their Resume.

**Describe the solution you'd like**
Making a component through which we can upload another image if don't want to use our GitHub profile picture.

**Describe alternatives you've considered**

**Additional context**
Perhaps this could help us out?
https://www.npmjs.com/package/react-images-upload

**Comment by ritvij14** [comment]
@garimasingh128 should we do this? I would like to give it a try.

**Comment by garimasingh128** [comment]
Yeah go ahead

**Comment by ritvij14** [comment]
@garimasingh128 I did more research on that npm package but couldn't figure out how it properly works. It itself is a component and is used more for sending data to the backend I think

That's why I added a form-input element. How does it look?

This is how the downloaded pdf image looks like. Good thing is that the form element isn't showing up in the pdf either.

The image is blurry because I think the quality of the pdf downloaded is not that good, as the text is also quite blurry.

![image](https://user-images.githubusercontent.com/51456744/93671235-e8854600-fabe-11ea-8b69-58028484c2ef.png)

This is on the website.

![image](https://user-images.githubusercontent.com/51456744/93671270-38fca380-fabf-11ea-9501-6dc594d1fc8d.png)
